### ISSUE TITLE
Changing settings does not work

### ISSUE THREAD

**Issue by Lomacar** [sdias/win-10-virtual-desktop-enhancer#59]
I have tried changing settings in the ini for keyboard shortcuts, tooltips, taskbar scrolling, desktop names, and wallpapers. The only settings that have any effect are the wallpapers.

**Comment by GioBonvi** [comment]
Hello @Lomacar,
Thank you for reporting this.

Have you restarted the program or reloaded the settings (right click on the tray icon -> reload settings) after changing the `settings.ini` file?

Are you sure you have downloaded a single copy of the program and are not editing the `settings.ini` file in another directory?

**Comment by Lomacar** [comment]
Yes, I was restarting the program. There is no "reload settings" option in the tray icon menu. I made sure I was editing the right settings.ini. I initially ran it from my downloads folder, and edited the wallpapers there. Then copied it to a permanent location and deleted the copy in the downloads folder.

**Comment by GioBonvi** [comment]
Ok, there is something strange going on here...

First of all you should be seeing this menu when right clicking on the tray menu icon of the program: it has been there since release [beta 0.8](https://github.com/sdias/win-10-virtual-desktop-enhancer/releases/tag/0.8) and there is really no reason for it not to be there in your system.
![image](https://user-images.githubusercontent.com/13332562/27422456-8300ad42-572e-11e7-80b7-3e252a53e664.png)

The only things I can think of are:

- Are you using the [latest version](https://github.com/sdias/win-10-virtual-desktop-enhancer/releases/latest)?
- Does the folder from where you are running the executable contain all the files needed? In particular all these files and folders should be in the same folder as shown in the image:

  - `virtual-desktop-enhancer.exe`
  - `settings.ini`
  - `libraries`
![image](https://user-images.githubusercontent.com/13332562/27422787-be078f18-572f-11e7-9d79-62df7c661e14.png)

- Have you tried deleting everything (folder and downloaded zip file) and downloading it again?
- Could you please copy here the contents of your `settings.ini` file?

**Comment by GioBonvi** [comment]
Hello @Lomacar,

sorry to bother you: are you still experiencing this issue?

**Comment by Lomacar** [comment]
Hi,

I must have downloaded the switcher from somewhere else. I noticed I didn't have the library folder, so I downloaded the repository, but the repository doesn't have the exe, then I went to the installation instructions and found the link to the "latest version", which I definitely never saw before. But I thought I had the latest version when I reported this bug. Anyway I downloaded the latest version and it doesn't work at all, but then I saw in the installation instructions that I need the Anniversary update or later. I am stuck on the original build of Windows 10. I live in a place with limited internet right now and I am waiting for a friend to mail me a copy of the Creator's Update.

**Comment by GioBonvi** [comment]
Hello @Lomacar,

while you are waiting for the Creator's Update you can try the [older releases](https://github.com/sdias/win-10-virtual-desktop-enhancer/releases): there will be some bugs and you will miss some of the features introduced later, but the program should work.

If I'm not wrong versions before 0.7 should run fine on a plain Windows 10 install, versions from 0.7 to (approximately) 0.10.4 should work if you have the Anniversary Update installed and the latest ones if you have Creator's Update.

In your case any version before 0.7 should work: you might also want to install [this](https://www.microsoft.com/en-us/download/details.aspx?id=52685) as suggested in the installation guide.

Best
